behat/tests/behat.yml

```diff
@@ -73,6 +73,12 @@ default:
             contexts:
                 - DigidepsBehat\Registration\RegistrationFeatureContext
 
+        userManagement:
```

Comment:
Others have been in format `user-management`, would be good to make them consistent.